tests/install/openqa_webui.pm

```diff
@@ -57,7 +57,8 @@ sub install_from_git {
     EOF
     script_run('env OPENQA_CONFIG=etc/openqa nohup script/openqa daemon &', 0);
     diag('Wait until the server is responsive');
-    assert_script_run('grep -qP "Listening at.*(127.0.0.1|localhost)" <(tail -F -n0 nohup.out) ', 600);
+    assert_script_run('grep -qP "Listening at.*(127.0.0.1|localhost)" <(tail -F nohup.out) ', 600);
+    get_log 'cat nohup.out' => 'openqa_nohup_out.txt';
```

Comment:
Why `script_run cat nohup.out` and then referring to the separate txt file instead of just

```
upload_logs 'nohup.out';
```

Reply:
I simply copied it from https://github.com/os-autoinst/os-autoinst-distri-openQA/blob/master/lib/openQAcoretest.pm#L18